openreview/conference/helpers.py

```diff
@@ -42,15 +42,24 @@ def get_conference_builder(client, request_form_id, support_user='OpenReview.net
         submission_start_date = None
 
     submission_due_date_str = 'TBD'
-    submission_due_date = note.content.get('Submission Deadline', '').strip()
-    if submission_due_date:
+    submission_second_due_date = note.content.get('Submission Deadline', '').strip()
```

Comment:
I think it is possible the user enter an abstract deadline but not a submission deadline, what should we do in that case?